R/calculateDMM.R

```diff
@@ -77,28 +77,19 @@
 #' tse <- TreeSummarizedExperiment(assays = list(counts = counts),
 #'                                 colData = colData)
 #'
+#' library(bluster)
+#' 
+#' # Compute DMM algorithm and store result in metadata
+#' tse <- cluster(tse, name = "DMM", DmmParam(),
```

Comment:
I would explicitly show the key DMM functions in the call (e.g. max clusters to test at least)